test_test.go

```diff
@@ -25,5 +29,27 @@ func TestGitCollector(t *testing.T) {
 	require.NoError(t, gitCollectorTest.Prepare())
 	require.NoError(t, gitCollectorTest.RunLoad())
 
-	gitCollectorTest.GetResults()
+	require.True(t, gitCollectorTest.GetResults())
+}
+
+// TestMetadataRetrieval
+// runs regression comparison for remote:master and the first release
+// <expected> no errors occurred during tests execution
+// <expected> GetResults returns true
+func TestMetadataRetrieval(t *testing.T) {
+	config := regression.NewConfig()
+	config.BinaryCache = "binaries"
+	config.Versions = []string{"remote:master", "v0.1.0"}
+	config.Repeat = 1
+	// No token, no access to teh v4 API
+	config.GitHubToken = os.Getenv("REG_TOKEN")
```

Comment:
I'm a bit confused about this part:
It probably does parse of the token with right permissions but you replace it with the token from env, that's not set in the CI or elsewhere

I have certainly missed something, could you please clarify it?

Reply:
I am doing this assignment because according to the README.md file you have to `export REG_TOKEN=your-github-token` for this project...so I am doing the assignment here in order to use it in metadataretrieval.go

```
command := NewCommand(metadataRetrieval.Path, orgs)
	start := time.Now()
	if err := command.Run(map[string]string{
		"LOG_LEVEL":     "debug",
		"GITHUB_TOKENS": t.config.GitHubToken,
	}); err != nil {
		t.log.With(log.Fields{
			"orgs":               orgs,
			"metadata-retrieval": metadataRetrieval.Path,
		}).Errorf(err, "Could not execute metadata-retrieval")
		return nil, err
	}
```

But you are right...it is not set in the .travis.yml, so I added it:

```
env:
  - GO111MODULE=on GOPROXY=https://proxy.golang.org
  - REG_TOKEN=$GITHUB_TOKEN
```

Reply:
ah, I see it comes from `regression-core`'s config...

***
aren't envs should be in one line? because afaik each envs line starts additional job
https://docs.travis-ci.com/user/environment-variables/#defining-multiple-variables-per-item

or am I missing something?

Reply:
Nope I have already changed it...tests are passing